src/main/java/Deadline.java

```diff
@@ -0,0 +1,23 @@
+public class Deadline extends Task {
```

Comment:
Missing Javadoc for all your files! May want to include Javadoc to tell the reader how the code works.